Sources/ApolloCodegenLib/Frontend/JavaScript/src/__tests__/deprecatedEnumCasesCompilationTests.ts

```diff
@@ -45,8 +45,7 @@ describe("given schema", () => {
     `;
 
     const document: DocumentNode = parseOperationDocument(
-      new Source(documentString, "Test Query", { line: 1, column: 1 }),
-      false
```

Comment:
You'll see a few of these changes removing the boolean value; it is the `experimentalClientControlledNullability` option which is not supported in graphql-js `17.0.0-alpha.2`.